flow/carrier_service/create.yaml

```diff
@@ -0,0 +1,20 @@
+triggers:
+  - keyword: create_shopify_carrier_service
+steps:
+  - ask: What name you want for your carrier service
+  - flow_set: carrier_service_name
+
+  - ask: So, do you want your carrier service to be named (@ flow.carrier_service_name)?
+    quick_replies:
+      - text: Yes
+        action: next
+      - text: No
+        action:
+          flow: flow.start
+          transfer: True
+
+  - type: meya.shopify.component.carrier_service.create
+    name: (@ flow.carrier_service_name)
+    integration: integration.shopify
+
+  - say: Carrier service sucessfully created!
```

Comment:
Switch to `**` here to get the GitHub Action to run: https://github.com/meya-customers/demo-ecommerce/blob/master/.github/workflows/meya.check-test.yaml#L8

(You should see `meya check` fail)